src/FloatingDockContainer.cpp

```diff
@@ -611,6 +611,8 @@ CFloatingDockContainer::CFloatingDockContainer(CDockManager *DockManager) :
     QDockWidget::setFeatures(QDockWidget::AllDockWidgetFeatures);
     setTitleBarWidget(d->TitleBar);
     connect(d->TitleBar, SIGNAL(closeRequested()), SLOT(close()));
+    connect(d->TitleBar, &CFloatingWidgetTitleBar::maximizeRequested,
+            this, &CFloatingDockContainer::onMaximizeRequest);

```

Comment:
Shouldn't there be a `ifndef Q_OS_LINUX` statement?  The `OnMaximizeRequest` is defined only for Linux, so here you will have a error anything except Linux.

Reply:
Good point. Not sure why the travis build succeeded.
Will change in a bit.

Reply:
Oh, it is. Look some lines up.

Reply:
Yeah you are right, I don't know how I missed it :D Sorry about that

Reply:
no worries